walletsetup.go

```diff
@@ -432,8 +432,13 @@ func createWallet(cfg *config) error {
 	netDir := networkDir(cfg.DataDir, activeNet.Params)
 	keystorePath := filepath.Join(netDir, keystore.Filename)
 	var legacyKeyStore *keystore.Store
-	if fileExists(keystorePath) {
-		var err error
+	_, err := os.Stat(keystorePath)
+	if err != nil && !os.IsNotExist(err) {
+		// A stat error not due to a non-existant file should be
+		// returned to the caller.
+		return err
+	} else if err == nil {
```

Comment:
@davecgh this one could be easily converted to use cfgutil.FileExists, however, since that error case should be returned here.